app/src/main/res/layout/fragment_inbox.xml

```diff
@@ -8,6 +8,8 @@
     app:theme="@style/AppBaseTheme"
     >
 
+
+
```

Comment:
codestyle

Reply:
fixed? (not sure applying codestyle changed anything in this class)